### PR TITLE
Change dependencies to point to the crates.io versions

### PR DESCRIPTION
**Pull request by Niederb**
Only for release to crates.io. 
Do not merge!

### CODE REVIEW

**Comment by Niederb** [comment]
@masapr Can you have a look at the meta-infos? I would then revert the dependency paths back to github and then we can actually merge this.

File: keystore/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# ac_keystore
+
+This crate is a submodule of the [substrate-api-client](https://github.com/scs/substrate-api-client). It contains a fork of the sc-keystore crate and this is in order to access private functions and members of the crate.
```
Comment (masapr) [inline keystore/README.md:3]: I would write the second sentence like this:

It is a fork of substrate's sc-keystore crate. The substrate-api-client needs its own fork in order to access private functions and members of the crate.

File: keystore/Cargo.toml [hunk 2]
```diff
@@ -12,11 +15,11 @@ parking_lot = "0.12.0"
...
-sc-keystore = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
-sp-application-crypto = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
-sp-core = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
-sp-keyring = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
-sp-keystore = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
+sc-keystore = { version = "27.0.0" }
+sp-application-crypto = { version = "32.0.0" }
+sp-core = { version = "30.0.0" }
+sp-keyring = { version = "33.0.0" }
+sp-keystore = { version = "0.36.0" }
```
Comment (masapr) [inline keystore/Cargo.toml:22]: This part we only need for the publishing, right? But we want to update the text below in general, is that right?
then this should be two PRs? Resp. only the PR for the text. And the dependencies here would only be adjusted for the branch that is published on crates.io.

**Review by masapr — CHANGES_REQUESTED** [review]

File: keystore/README.md (hunk 1 above)
Comment (Niederb) [reply keystore/README.md]: I split the PR into two and added your suggestion to the new PR. See #757

File: keystore/Cargo.toml (hunk 2 above)
Comment (Niederb) [reply keystore/Cargo.toml:21]: Opened a separate PR. See https://github.com/scs/substrate-api-client/pull/757

**Comment by Niederb** [comment]
Changes are only needed to release to crates.io. Closing this

**Pull request closed without merging**